collector/xfs_linux.go

```diff
@@ -199,6 +199,41 @@ func (c *xfsCollector) updateXFSStats(ch chan<- prometheus.Metric, s *xfs.Stats)
 			desc:  "Number of times the directory getdents operation was performed for a filesystem.",
 			value: float64(s.DirectoryOperation.Getdents),
 		},
+		{
+			name:  "inode_operation_attempts_total",
+			desc:  "Number of times the OS looked for an XFS inode in the inode cache.",
+			value: float64(s.InodeOperation.Attempts),
+		},
+		{
+			name:  "inode_operation_found_total",
+			desc:  "Number of times the OS looked for and found an XFS inode in the inode cache.",
+			value: float64(s.InodeOperation.Found),
+		},
+		{
+			name:  "inode_operation_recycle_total",
+			desc:  "Number of times the OS found an XFS inode in the cache, but could not use it as it was being recycled.",
+			value: float64(s.InodeOperation.Recycle),
+		},
+		{
+			name:  "inode_operation_missed_total",
+			desc:  "Number of times the OS looked for an XFS inode in the cache, but did not find it.",
+			value: float64(s.InodeOperation.Missed),
+		},
+		{
+			name:  "inode_operation_duplicate_total",
+			desc:  "Number of times the OS tried to add a missing XFS inode to the inode cache, but found it had already been added by another process.",
+			value: float64(s.InodeOperation.Duplicate),
+		},
+		{
+			name:  "inode_operation_reclaims_total",
+			desc:  "Number of times the OS reclaimed an XFS inode from the inode cache to free memory for another purpose.",
+			value: float64(s.InodeOperation.Reclaims),
+		},
+		{
+			name:  "inode_operation_attrchange_total",
```

Comment:
Let's avoid abbreviations.

```suggestion
			name:  "inode_operation_attribute_changes_total",
```

Reply:
Done.  I've also made the metrics consistently plural.   Note that in the xfs.Stats struct (https://github.com/prometheus/procfs/blob/aa47f9d89cea5aaf43204594c92a9a8460544388/xfs/xfs.go#L105) the names are not consistently pluralized and I had made the metrics conform to that.

Reply:
Thanks! Yea, I wasn't sure what is the canonical correctness. Either way, the underlying struct doesn't matter as much as having it consistent on the metric naming side.